<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterEdit_base</class>
 <widget class="QWidget" name="FilterEdit_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>443</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="KListWidgetSearchLine" name="searchline"/>
         </item>
         <item>
          <widget class="QListWidget" name="mRulesList"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="mNewButton">
           <property name="text">
            <string>&amp;New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mDeleteButton">
           <property name="text">
            <string>&amp;Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="mDetailsFrame">
       <property name="title">
        <string>Filter Details</string>
       </property>
       <layout class="QGridLayout">
        <property name="margin">
         <number>11</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1">
            <property name="text">
             <string comment="filter name">Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KLineEdit" name="mNameLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="mRecurringCheck">
          <property name="whatsThis">
           <string>Select this option if you do not want to show recurring events and to-dos in your views. Daily and weekly recurring items may take a lot of space, so it might be handy to hide them.</string>
          </property>
          <property name="text">
           <string>Hide &amp;recurring events and to-dos</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="mCompletedCheck">
          <property name="whatsThis">
           <string>If this box is checked, the filter will hide all to-do items from the list, that have been completed. Optionally, only items that have been completed a given number of days are hidden.</string>
          </property>
          <property name="text">
           <string>Hide co&amp;mpleted to-dos</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="mCompletedTimeSpanLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="whatsThis">
             <string>This option will allow you to select which completed to-dos should be hidden. When you choose &lt;i&gt;Immediately&lt;/i&gt;, it will hide the to-do as soon as you check it. You can increase or decrease the number of days in the spinbox.</string>
            </property>
            <property name="text">
             <string>Days after completion:</string>
            </property>
            <property name="buddy">
             <cstring>mCompletedTimeSpan</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KIntNumInput" name="mCompletedTimeSpan">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="whatsThis">
             <string>Here you can give the number of days a to-do item has to be completed to be hidden from the to-do list. If you select &quot;Immediately&quot;, all completed to-dos will be hidden. If you, for example, choose a value of 1, all to-do items will be hidden, that have been marked finished longer than 24 hours ago.</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>16</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="mHideInactiveTodosCheck">
          <property name="whatsThis">
           <string>This option hides all to-dos from your list, where the start date has not been reached. (Note that the start date is not the due date of the to-do item.)</string>
          </property>
          <property name="text">
           <string>Hide &amp;inactive to-dos</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QGroupBox" name="mCategoriesButtonGroup">
          <property name="title">
           <string>Categories</string>
          </property>
          <layout class="QGridLayout">
           <property name="margin">
            <number>11</number>
           </property>
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="2" column="0" rowspan="2">
            <widget class="QListWidget" name="mCatList">
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <spacer>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>110</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QRadioButton" name="mCatHideCheck">
             <property name="whatsThis">
              <string>When this option is enabled, this filter will show all items which do &lt;i&gt;not&lt;/i&gt; contain the selected categories.</string>
             </property>
             <property name="text">
              <string>Show all except selected</string>
             </property>
             <property name="shortcut">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QRadioButton" name="mCatShowCheck">
             <property name="whatsThis">
              <string>When this option is enabled, this filter will show all items containing at least the selected items.</string>
             </property>
             <property name="text">
              <string>Show only selected</string>
             </property>
             <property name="shortcut">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="mCatEditButton">
             <property name="text">
              <string>Change...</string>
             </property>
             <property name="shortcut">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="mHideTodosNotAssignedToMeCheck">
          <property name="whatsThis">
           <string>This option hides all to-dos from your list which are assigned to someone else.&lt;br&gt;
Only to-dos which have least one attendee will be checked.  If you are not in the list of attendees the to-do will be hidden.</string>
          </property>
          <property name="text">
           <string>Hide to-dos not assigned to me</string>
          </property>
          <property name="shortcut">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>mRulesList</zorder>
  <zorder>mNewButton</zorder>
  <zorder>mDeleteButton</zorder>
  <zorder>mDetailsFrame</zorder>
  <zorder>klistwidgetsearchline</zorder>
  <zorder>mRulesList</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidgetSearchLine</class>
   <extends>KLineEdit</extends>
   <header>klistwidgetsearchline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
